scripts/editor/get-options.js

```diff
@@ -6,10 +6,12 @@ import _ from 'lodash';
  *
  * @author: Kristijan Marić <kristijan.maric@infinum.co>
```

Comment:
Maybe remove the email address since it's no longer valid?

Reply:
```suggestion
 * @author: Kristijan Marić
```